_posts/guides/2019-08-23-guide_install-css-assets.md

```diff
@@ -7,4 +7,32 @@ tags:
   - cs:s
 ---
 
-TODO: Show how to install CS:S assets from SteamCMD
+
+This Guide covers how to install CS:S assets from SteamCMD.
+## Prerequisites 
+- An internet connection.
+- some free space on your pc. (~20GB)
+
+## Getting SteamCMD
+
+- Make a folder for SteamCMD.
+- Inside of it download [***SteamCMD***](https://steamcdn-a.akamaihd.net/client/installer/steamcmd.zip), and unzip it. 
+- Start steamcmd.exe and wait until you got 'Steam>' waiting for your input.
```

Comment:
This is assuming Windows. Can just link to Valve's guide for Linux/Mac: https://developer.valvesoftware.com/wiki/SteamCMD